.github/workflows/acceptance.yml

```diff
@@ -31,7 +31,8 @@ jobs:
     # Map a step output to a job output
     outputs:
       acceptance: ${{ steps.changes.outputs.acceptance }}
-      backend: ${{ steps.changes.outputs.backend_all }}
+      backend_all: ${{ steps.changes.outputs.backend_all }}
+      frontend_all: ${{ steps.changes.outputs.frontend_all }}
```

Comment:
`frontend_all` was only added in the last week, thus, the drift was not as bad
https://github.com/getsentry/sentry/blob/b9ed360540ae6d31bdc827a500ec8f761ead5fd4/.github/workflows/acceptance.yml#L126